Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -377,12 +377,19 @@ public struct ConvertAction: Action, RecreatingContext {
         }
 
         // Process Static Hosting as needed.
-        if transformForStaticHosting, let templateDirectory = htmlTemplateDirectory {
+        if transformForStaticHosting,
+           let templateDirectory = htmlTemplateDirectory,
+           // If this conversion didn't actually produce documentation, then we expect
+           // the creation of this data provider to file because there will be no 'data' subdirectory
+           // in the documentation output. (r91790147)
+           let dataProvider = try? LocalFileSystemDataProvider(
```

Comment:
This looks good to me. `LocalFileSystemDataProvider` only raises errors related to reading the content of directories and files so I don't think we shouldn't be missing any important errors here.